soopercool/utils.py

```diff
@@ -157,6 +123,39 @@ def random_src_mask(mask, nsrcs, mask_radius_arcmin):
     return ps_mask
 
 
+def get_beam_windows(meta, plot=False, beam_floor=1.e-2):
```

Comment:
Is beam_floor actually used here?

Reply:
No. I will remove it. Beam floor is applied when loading the beams.